test/test-utils/helpers/overridesHelper.ts

```diff
@@ -188,12 +189,29 @@ export class OverridesHelper extends GameManagerHelper {
   }
 
   /**
-   * Override each wave to not have random trainer battles
+   * Override the chance of encountering a random enemy trainer
+   * @param trainerChance - `0` to disable enemy trainer spawns, `1` to guarantee an enemy trainer spawn
+   * @see {@linkcode Overrides.RANDOM_TRAINER_CHANCE_OVERRIDE} for a more complete description of this override
+   * @returns `this`
+   */
+  public trainerChance(trainerChance: number): this {
+    vi.spyOn(Overrides, "RANDOM_TRAINER_CHANCE_OVERRIDE", "get").mockReturnValue(trainerChance);
+    if (trainerChance === 0 || trainerChance === 1) {
+      this.log(`Trainer chance override set to ${trainerChance}!`);
+    } else {
+      this.log(`Trainer chance override set to 1 / ${trainerChance}!`);
```

Comment:
If you still want the "1 / " to not show up when `trainerChance === 1`:
```suggestion
      this.log(`Trainer chance override set to ${trainerChance > 1 ? "1 / " : ""}${trainerChance}!`);
```